EscapeButton/EscapeButton/Form1.cs

```diff
@@ -0,0 +1,35 @@
+// <copyright file="Form1.cs" company="Sarapulov Vasiliy">
+// Copyright (c) Sarapuliv Vasiliy. All rights reserved.
+// </copyright>
+
+namespace EscapeButton
+{
+    /// <summary>
+    /// A class that implements a form.
+    /// </summary>
+    public partial class Form1 : Form
+    {
+        private readonly Random rnd = new Random();
```

Comment:
```suggestion
        private readonly Random rnd = new();
```
А ещё имена сокращать нельзя.